examples/client-logger/app-insights-logger/jest.config.js

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
```

Comment:
I would recommend mimicking the test pattern used in our other example apps - that way you can get (at least) a smoke test up and running for the app. `external-data` might be a good point of reference, though it does some extra server setup steps that won't be needed here.

Reply:
I spent all day trying to get an integration test to work and failed to so.. But I build a much simpler test and confirms it renders.